src/components/radio-filter/radio-filter.jsx

```diff
@@ -23,6 +24,13 @@ export default React.createClass({
 
     this.props.onChange(choice);
   },
+  componentWillReceiveProps(nextProps){
+    if (nextProps.value !== this.state.activeFilter){
+      this.setState({
+        activeFilter: nextProps.value
+      });
+    }
+  },
```

Comment:
Actually this doesnot seem correct see http://stackoverflow.com/questions/32414308/updating-state-on-props-change-in-react-form